client/me/account/index.jsx

```diff
@@ -80,18 +90,22 @@ module.exports = React.createClass( {
 			} else {
 				this.setState( { redirect: false } );
 			}
-		}.bind( this );
+		};
 
 		return valueLink;
 	},
 
-	validateUsername: function() {
-		var username = this.props.userSettings.getSetting( 'user_login' );
+	validateUsername() {
+		const username = this.props.userSettings.getSetting( 'user_login' );
 		debug( 'Validating username ' + username );
 		this.props.username.validate( username );
 	},
 
-	communityTranslator: function() {
+	hadFormError() {
+		return this.state.emailValidationError;
```

Comment:
Since we store a string in `this.state.emailValidationError`, we may need to return `!! this.state.emailValidationError` to make sure we return a `boolean`.